wait/member.go

```diff
@@ -84,6 +88,7 @@ func (a *MemberAwaitility) WaitForDeletedNSTmplSet(name string) error {
 	})
 }
 
+// GetNamespace returns the namespace of the given type and owned by the given username
 func (a *MemberAwaitility) GetNamespace(username, typeName string) *v1.Namespace {
```

Comment:
why didn't you replace this method by wait too?

Reply:
because I forgot, I guess 😂

Reply:
done in https://github.com/codeready-toolchain/toolchain-e2e/pull/31/commits/fb61d38ddb23e008bb010262eb8b44be266a39cf